### PR TITLE
Serialize JS code events and extension descriptions as arrays of strings to make collaboration/reviews easier

### PR DESCRIPTION
**Pull request by 4ian**
Only show in developer changelog

### CODE REVIEW

**Review by ClementPasteau — APPROVED** [review]
Looks pretty cool to me.
I assume there's no impact on existing projects?

**Comment by 4ian** [comment]
Good question! Existing project will be opened as usual, and when saved the JS code events `inlineCode`, and the `description` of extensions, will be saved as arrays. 
If you open the game in a previous version, this will be read as an empty string so games won't be backward compatible.

**Comment by 4ian** [comment]
We actually need to change the GDevelop-extensions deployment step so that any description and any JS code event is serialised to a string, so we keep compatibility with the existing versions of GDevelop. We can then remove this compatibility code once we have enough people using the last version.

Otherwise, extensions won't have descriptions and JS code events won't work

File: Core/GDCore/Project/EventsFunctionsExtension.cpp [hunk 1]
```diff
@@ -54,7 +54,11 @@ void EventsFunctionsExtension::SerializeTo(SerializerElement& element) const {
   element.SetAttribute("version", version);
   element.SetAttribute("extensionNamespace", extensionNamespace);
   element.SetAttribute("shortDescription", shortDescription);
+
+  // TODO Use the multi line serialization in a release that follows the 150.
+  // element.AddChild("description").SetMultilineStringValue(description);
   element.SetAttribute("description", description);
```
Comment (D8H) [inline Core/GDCore/Project/EventsFunctionsExtension.cpp:60]: I made this change.
I'm not use if this is what you had in mind.

**Pull request merged**